packages/tools/devtools/devtools-view/src/components/TelemetryView.tsx

```diff
@@ -421,6 +440,12 @@ function FilteredTelemetryView(props: FilteredTelemetryViewProps): React.ReactEl
 		if (matchingOption) {
 			const search = data.optionText !== undefined ? data.optionText : "";
 			setCustomSearch(search);
+			usageLogger?.sendTelemetryEvent({
+				eventName: "TelemetryEventNameFilter",
+				details: {
+					eventNames: [search],
+				},
```

Comment:
Remove details; similar to other bits we removed, this is entirely user-generated data so we shouldn't log it.